kafka-ui-react-app/src/components/Topics/New/CustomParams/CustomParamValue.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useFormContext, ErrorMessage } from 'react-hook-form';
+import { CUSTOM_PARAMS_OPTIONS } from './customParamsOptions';
+import { isFirstParam } from './CustomParams';
+
+interface Props {
+  isDisabled: boolean;
+  index: string;
+  name: string;
+  defaultValue: string;
+}
+
+const CustomParamValue: React.FC<Props> = ({
+  isDisabled,
+  index,
+  name,
+  defaultValue,
+}) => {
+
+  const { register, unregister, errors, watch, setValue } = useFormContext();
+  const selectInputName: string = `${index}[name]`;
+  const valInputName: string    = `${index}[value]`;
+  const selectedParamName: any  = watch(selectInputName, name);
```

Comment:
any?

Reply:
I'm a typescript newbie =) Not sure how to handle this if things like `watch()`, `getValues()` return `undefined`

which is ok?
`const selectedParamName = watch(selectInputName, name);` 
`const selectedParamName: string = watch(selectInputName, name);` 
but if I do this way, I start getting typescript warnings like: Type 'uknown' is not assignable to type 'string' or Type 'unknown' cannot be used as an index type.

Please do suggest a correct option